tesseract/tesseract_planning/tesseract_process_managers/examples/freespace_manager_example.cpp

```diff
@@ -4,13 +4,14 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <taskflow/taskflow.hpp>
 #include <fstream>
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
```

Comment:
The examples are missing licenses. I am fine with Apache 2 for any parts I may have contributed.

Reply:
Do you know which examples you added?

Reply:
No. I don't really care. I just wanted to put it on the internet that I was ok with the license in case people saw that you were adding licenses to existing code.

Reply:
Ok.